packages/react/src/Link/Link.tsx

```diff
@@ -27,17 +27,22 @@ const hoverColor = system({
 const StyledLink = styled.a<StyledLinkProps>`
   color: ${props => (props.muted ? get('colors.fg.muted')(props) : get('colors.accent.fg')(props))};
 
-  /* By default, Link does not have underline */
+  /* By default, Link is not underlined. */
   text-decoration: none;
 
-  /* You can add one by setting underline={true} */
+  /* If inline or underline are set to true Link is underlined. */
   text-decoration: ${props => (props.underline ? 'underline' : undefined)};
+  text-decoration: ${props => (props.inline ? 'underline' : undefined)};
```

Comment:
I think we'll need one line otherwise the `inline` value will always override, maybe something like:

```suggestion
  text-decoration: ${props => (props.underline || props.inline ? 'underline' : undefined)};
```

Reply:
Thank you, it's not a problem but definitely better style. Remaining problem is the data attribute if set to `false` affecting the `underline: true`. We might want to deprecate `underline` first 🤔